### ISSUE TITLE
[ENHANCEMENT] Value setter

### ISSUE THREAD

**Issue by otavlna** [Applifting/nestjs-decorated-config#5]
Sometimes we receive the value of variable from `Config` in runtime.

For example, we can have an integration test which spins up its database container, and only after it is spun, we receive its PORT. There is no way to set the value of `config.databasePort`.

**Comment by otavlna** [comment]
Actually this is out of scope of this package, `Config` is defined in app
